.github/workflows/azure-sdk-tools.yml

```diff
@@ -0,0 +1,33 @@
+name: Test Azure SDK Tools
```

Comment:
At first, I thought you were adding a workflow to build our azure-sdk-tools repo... :)